bash/scripts/find_required_commits.sh

```diff
@@ -4,8 +4,8 @@ set -eo pipefail
 # find-required-commits parses a commit description for any required sibling
 # commits to pass along to downstream job(s)...
 find-required-commits() {
-  local git_commit="${1}"
-  local commit_description="${COMMIT_DESCRIPTION:-$(git log --format=%B -n 1 "${git_commit}")}"
+  git_commit="${1}"
```

Comment:
Yeah I'm familiar with that behavior. Using local clobbers the return code of the execution of a subshell or the setting of the variable. So if you're relying on grabbing the return code after setting the variable with the command's output you can't prefix the line with local. However, if the subshell errors out, the script will still fail due to the `set -eo pipefail`.

If this solves an actual problem... cool. But I'm still intrigued by what actually might be happening.